### PR TITLE
Support l2Pool2d

### LINKED ISSUE #18: Implement l2Pool2d
Spec: https://webmachinelearning.github.io/webnn/#api-neuralnetworkcontext-pool2d

### PR DESCRIPTION
**Pull request by BruceDai**
This pr is to fix #18. Since tf.js doesn't support l2Pool2d, I implement l2Pool2d with following workaround in pool2d.ts:
```ts
if (poolingType === 'l2') {
      input = tf.pow(input, 2);
      output = tf.sqrt(
        tf.pool(input, this.windowDimensions_, 'avg', padding, this.dilations_,
        this.strides_));
}
```
 
@huningxin PTAL, thanks.

### CODE REVIEW

File: src/nn/graph_builder.ts [hunk 1]
```diff
@@ -574,6 +574,14 @@ export class MLGraphBuilder {
     return (new AveragePool2d(input, options)).output;
   }
 
+  /**
+   * [spec](https://webmachinelearning.github.io/webnn/#dom-mlgraphbuilder-l2pool2d)
+   */
+   l2Pool2d(input: MLOperand, options: MLPooling2dOptions = {}): MLOperand {
```
Comment (huningxin) [inline src/nn/graph_builder.ts:580]: Please check the indent of this line.

**Review by huningxin — APPROVED** [review]
LGTM with a nit. Thanks much @BruceDai .

**Comment by BruceDai** [comment]
Thanks @huningxin . I've fixed indent error, please take another look, thanks.

And about CI failure I filed #114 to track and also reported https://github.com/tensorflow/tfjs/issues/5566 for tensorflow/tfjs. 

Should we workaround with use tfjs 3.8.0  as I submitted the last commit before tensorflow fix https://github.com/tensorflow/tfjs/issues/5566?

**Review by huningxin — APPROVED** [review]
LGTM, thanks much @BruceDai .

**Pull request merged**